scripts/establish_conda_env.sh

```diff
@@ -232,13 +253,26 @@ do
   shift
 done
 
+# get installation manager
+INSTALL_MANAGER="$(echo $INSTALL_MANAGER | tr '[a-z]' '[A-Z]')"
+echo $INSTALL_MANAGER
+if [[ "$INSTALL_MANAGER" != "CONDA" && "$INSTALL_MANAGER" != "PIP" ]];
+then
+  echo ... ERROR: installation-manager $INSTALL_MANAGER UNKNOWN. Available are [CONDA, PIP] !
+  exit
+fi
+
 if [[ $ECE_VERBOSE == 0 ]];
 then
   echo ... Run Options:
   echo ...    Mode: $ECE_MODE
   echo ...   Verbosity: $ECE_VERBOSE
   echo ...   Clean: $ECE_CLEAN
-  echo ...   Conda Defs: $CONDA_DEFS
+  echo ...    Mode: $INSTALL_MANAGER
+  if [[ "$INSTALL_MANAGER" != "CONDA" ]];
+  then
+    echo ...   Conda Defs: $CONDA_DEFS
```

Comment:
why not "conda" then echo the conda_defs?

Reply:
right.